### ISSUE TITLE
Support for multiple api keys

### ISSUE THREAD

**Issue by jerry-enebeli** [blnkfinance/blnk#29]

**Comment by ankiiisharma** [comment]
hey @jerry-enebeli , I would like to lwork on this. can you elaborate this more?
thanks

**Comment by jerry-enebeli** [comment]
Hi @ankiiisharma Thanks for your interest in the Issue. I'll update it with a detailed description in a few days(it should be up before Friday next week). If it's something you still feel like working on, you can send in a PR, and I’ll gladly review it.

**Comment by ankiiisharma** [comment]
Yes sure, please update the description i will look into this. Thanks
